pages/requiredfields.js

```diff
@@ -1436,8 +1475,11 @@ function RequiredFields(props) {
                         as="textarea"
                         rows={3}
                         placeholder="Enter content"
-                        value={content}
-                        onChange={(e) => setContent(e.target.value)}
+                        // value={userContent}
```

Comment:
You do need this value uncommented. This is necessary displaying the value. 

Every time the user types on their keyboard, the `onChange` method is called. Lets say the following situation happens. Currently, the textarea is empty, and I want to type "Hello"

With the current implementation, when I enter "H", we will call `setUserContent("H")`. But, because the value is never set, it will remain a blank string so the user won't see the value typed in the text box.